profile.html

```diff
@@ -0,0 +1,154 @@
+                </button>
+                <div class="collapse navbar-collapse ms-lg-5" id="navbarNav">
+                    <ul class="navbar-nav">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="profile.html">Profile</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="index.html">Feed</a>
+                        </li>
+                    </ul>
+                    <div class="container-fluid d-flex justify-content-lg-end">
+                        <form class="d-flex" role="search">
+                            <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                            <button class="btn btn-outline-success" type="submit">Search</button>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="card mb-3 m-5 border-3 border-primary">
+            <div class="row g-0">
+                <div class="col-md-4">
+                    <img src="/dist/images/avatar1.png" class="img-fluid rounded-start img-thumbnail" alt="...">
+                </div>
+                <div class="col-md-8">
+                    <div class="card-body">
+                        <h2 class="card-title">Jimmy Lakers</h2>
+                        <p class="card-text">Y'all gon' make me lose my mind<br>
+                        Up in here, up in here<br>
+                        Y'all gon' make me go all out<br>
+                        Up in here, up in here.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="container d-flex flex-column align-items-center text-center">
+            <form action="/profile.html" class="mt-5">
+                <h2 class="h5">Wallpost:</h2>
+                <div class="mb-3">
+                    <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                </div>
+                <button class="btn btn-primary btn-lg">Post</button>
+            </form>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend4.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
+                <div class="card-body">
+                    <p class="card-text">Happy birthday, you son of a gun!</p>
+                        <a href="#" class="btn btn-primary">Reply</a>
+                </div>
+            </div>
+            <span class="m-3"></span>
+            <div class="card d-flex flex-column align-items-center text-center" style="width: 25rem;">
+                <img src="/dist/images/friend5.png" class="card-img-top img-thumbnail m-2 img-height img-width" alt="...">
```

Comment:
Why do you have images in the dist folder? Static files such as images, favicons, fonts, etc are meant to be kept in public. So I suggest you do:

- create a public folder in the root folder
- move the images folder in the dist folder into the public folder
- fix the path in diff places in your html files

Reply:
Not sure what I was thinking placing them in dist. Fixed as suggested.